charts/gateway/README.md

```diff
@@ -439,6 +439,7 @@ The following table lists the configurable parameters of the Gateway chart and t
 | `global.schedulerName`                      | Override the default scheduler | `nil` |
 | `license.value`          | Gateway license file | `nil`  |
 | `license.accept`          | Accept Gateway license EULA | `false`  |
+| `disklessConfig`          | Boolean value whether to use diskless configuration for Gateway or not. When true, environment variables are used to configure Gateway. When false, node.properties is used. | `true` |
```

Comment:
@jennarddy , Can we say " Diskless Config Flag , whether to use ..."

Reply:
i wonder if we want to hide this. It is hard to properly illustrate what this actually does and may create confusion. 

suggestion: keep the variable as is, remove it from this table. Rely on techdoc to properly show how to configure helm-side

Reply:
It might be better to just include a link to Techdocs for that field for further reading. We can do that in this table as well as the Diskless Config section